### DISCUSSION TITLE
“code .” command fails with “Your browser doesn’t support opening local folders”

### DISCUSSION THREAD

**Discussion by oplatek** [blinksh/blink#1631] [Support] [answered]
I followed the video at https://youtu.be/q-sBdfC9DL8

Using blink v16.0.1.616 on iPad OS 16.1.1

1. I opened a new blink window
2. Typed `code .`
3. The VSCode window appeared
![Screenshot 2022-11-11 at 12 09 39](https://user-images.githubusercontent.com/229266/201329746-bfeaf10a-1110-47e2-80b3-6253be9de77f.png)

I do not know how to edit the local files 😂 

Any help appreciated

**Comment by carloscabanero** [comment, marked as answer]
Have you installed the Blink FS extension? https://docs.blink.sh/advanced/code

**Reply by oplatek** [reply]
Nope. After I have done it is fixed :) 

Thanks!

**Comment by oplatek** [comment]
BUT before I installed I was not able to type in the Extensions window 😄 

So I needed to sort the extensions according to downloads and look up Blink FS manually 😛
